app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -416,13 +417,26 @@ open class DefaultToolbarMenu(
         context.getString(R.string.sync_menu_sign_in)
     }
 
-    val syncedTabs = BrowserMenuImageText(
-        syncItemTitle,
-        R.drawable.ic_synced_tabs,
-        primaryTextColor()
-    ) {
-        onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)
-    }
+    val syncTabsOrSignInItem =
+        if (tabsTrayRewrite) {
+            // if synced tabs are being shown in tabs tray, show sync sign in here.
```

Comment:
Sorry for this minor nit

```suggestion
            // If synced tabs are being shown in tabs tray, show sync sign in here.
```

Reply:
😂